configure.ac

```diff
@@ -1224,6 +1224,23 @@ AM_CONDITIONAL([WRAPPABLE_UNAME], [test x"$WRAPPABLE_UNAME" = x"yes"])
 
 cc_restore_flags
 
+dnl ========================================================================
+dnl Byte size
+dnl ========================================================================
+
+# Compile-time assert hack
+# https://jonjagger.blogspot.com/2017/07/compile-time-assertions-in-c.html
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <limits.h>]],
+                                   [[
+                                        switch (0) {
+                                            case 0:
+                                            case (CHAR_BIT == 8):
+                                                break;
+                                        }
+                                   ]])],
+                  [],
+                  [AC_MSG_FAILURE("char must be 8 bits")])
```

Comment:
being a little pedantic, I would say something like "Pacemaker does not support platforms where char is not 8 bits" (to acknowledge the burden is on us, not that the hardware is at fault)

Reply:
That is better